nifi-frontend/src/main/frontend/apps/nifi/src/app/pages/users/service/users.service.ts

```diff
@@ -73,7 +71,7 @@ export class UsersService {
                 ...request.userPayload
             }
         };
-        return this.httpClient.put(this.nifiCommon.stripProtocol(request.uri), payload);
+        return this.httpClient.put(`${UsersService.API}/tenants/users/${request.id}`, payload);
```

Comment:
I believe the `uri` field in the `request` object is no longer used and can be removed.